skrf/calibration/deembedding.py

```diff
@@ -0,0 +1,80 @@
+from ..frequency import *
+from ..network import *
+

```

Comment:
tests seems to fail because of missing  `abstractmethod`
```suggestion
from abc import ABC, abstractmethod
```

Reply:
I'm adopting `ABC` and `abstractmethod` into this de-embedding class. So hopefully it goes away.